data/json/professions.json

```diff
@@ -5187,5 +5187,50 @@
                 "boy_shorts"
             ]
         }
+    },
+    {
+        "type": "profession",
```

Comment:
@mugling I could move some Professions into More Professions in this PR; do you have any suggestions?